classes/controller/FrontController.php

```diff
@@ -80,6 +80,7 @@ class FrontControllerCore extends Controller
     /**
      * If set to true, user can be logged in as guest when checking if logged in.
      *
+     * @deprecated Since 8.0 and will be removed in the next major.
```

Comment:
```suggestion
     * @deprecated Since 8.1 and will be removed in the next major.
```

Reply:
The version is correct, I am targeting 8.0 branch

Reply:
Deprecating something in a patch version should not be allowed IMO :/ I did not check the targeted branch.

Reply:
I can remove the deprecation notice and add it in 8.1, but it will have the same effect. Worse, because people will find out later.

Anyway, this should be merged ASAP to 8.0.2